### PR TITLE
doc: Add release notes for pr18331

### PR DESCRIPTION
**Pull request by hebasto**
Add release notes for #18331.

### CODE REVIEW

File: doc/release-notes-18331.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Build System
+------------
+
+The source code archives that are provided with gitian builds do not contain
```
Comment (ryanofsky) [inline doc/release-notes-18331.md]: Would s/do not contain/no longer contain/ to clarify

File: doc/release-notes-18331.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+any autotools artifacts. Therefore, to build from such source a user 
+should run the `.autogen.sh` script from the root of the unpacked archive.
```
Comment (ryanofsky) [inline doc/release-notes-18331.md]: Missing slash in ./autogen.sh

**Review by ryanofsky — APPROVED** [review]
ACK 2d0d540759eaeddead2ecd71ae642851a67c9d5a. Thanks for following up!

File: doc/release-notes-18331.md (hunk 2 above)
Comment (hebasto) [reply doc/release-notes-18331.md]: Fixed.

File: doc/release-notes-18331.md (hunk 1 above)
Comment (hebasto) [reply doc/release-notes-18331.md]: Fixed.

**Comment by MarcoFalke** [comment]
Should go to the wiki, not here

**Comment by MarcoFalke** [comment]
https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.20.0-Release-Notes-Draft

**Comment by hebasto** [comment]
> Should go to the wiki

Done.

**Comment by ryanofsky** [comment]
Thanks! https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.20.0-Release-Notes-Draft#build-system LGTM

**Pull request closed without merging**